Bootstrapper1/bootstrapper1.wxs

```diff
@@ -1,16 +1,12 @@
-<?xml version="1.0" encoding="UTF-8"?>
-<Wix xmlns="http://schemas.microsoft.com/wix/2006/wi" xmlns:bal="http://schemas.microsoft.com/wix/BalExtension" xmlns:util="http://schemas.microsoft.com/wix/UtilExtension">
```

Comment:
Fix: Rename http://schemas.microsoft.com/wix/BalExtension to http://wixtoolset.org/schemas/v4/wxs/bal